athena-postgresql/athena-postgresql.yaml

```diff
@@ -0,0 +1,103 @@
+Transform: 'AWS::Serverless-2016-10-31'
+Metadata:
+  'AWS::ServerlessRepo::Application':
+    Name: AthenaPostGreSqlConnector
```

Comment:
nit: Does this have to be consistent with how our classes use `PostGres` ? Asking because this is just how the application is gonna be labeled in SAR

Reply:
Not really, I'm unopinionated about the naming here. it'll just be confusing when the connector is named one thing and the handler class parameter is named another thing.